rrrspec-client/lib/rrrspec/client/rspec_runner_factory.rb

```diff
@@ -0,0 +1,16 @@
+require 'rrrspec/client/rspec_runner'
+
+module RRRSpec
+  module Client
+    class RSpecRunnerFactory
+
+      def initialize
+        @runner = RSpecRunner.new
+      end
+
+      def create(file_extension)
+        return @runner if file_extension == '.rb'
```

Comment:
Removed the factories as the they don't make sense having the registration by file extension.